Improved download page

msemple1111 opened the pull request:
Improved the download page like it is [suggested here](https://github.com/Upload/Up1/issues/14)

k3d3 commented on index.html:
```diff
@@ -150,9 +150,13 @@
-            font-size: 30px;
+        }
+
+        .linebrBefore:before {
```
What's the purpose of this class? Could this be solved using padding or margin instead?

msemple1111 replied:
It should have been solved that way. I have corrected it to use margin instead now.

k3d3 commented on static/js/download.js:
```diff
@@ -169,7 +169,11 @@ upload.modules.addmodule({
+            $('<h1>').text(data.header.name).appendTo(dlarea)
+            $('<p>').text('No preview available').appendTo(dlarea)
+			var bigdlbtn = $('<a>').addClass('bigbtn').prop('href', url).appendTo(dlarea)
+			$('<a>').addClass('linebrBefore').text('Download').appendTo(bigdlbtn)
```
Is this adding an `<a>` tag inside another `<a>` tag?

msemple1111 replied:
It was adding an `<a>` tag inside an `<a>` tag. My mistake sorry - i have corrected it now.

andre-d commented:
Aside from the code style and code problems, the UI is a bit confusing.  Re-using the upload dialog is confusing as previously that sort of UI is a drop zone for files and now it is a download button so it seems weird as a pure button.  But aside from that small confusion the UI itself looks bad because the large box is not centered vertically and its hard for the eye to find a space.  It would be better with a smaller download button and if you really want a box putting a border around the whole thing.

msemple1111 commented:
I updated the UI according to your suggestions @andre-d. I'm not particularly good at UI design so welcome any suggestions! 
I have fixed the code problems you mentioned @k3d3. I think that is it but if you find a few more, i'll gladly fix them too.

yuu0007 commented:
hello
